Updated README with realm-cli push instructions

therealmark opened the pull request:
Importing the existing production-app into Atlas is not possible through the Atlas UI. So we had to update the README with instructions on how to install the realm-cli and use it to "push" the production-app to Atlas.

therealmark committed:
updated readme with instructions on how to deploy the App Services App

therealmark committed:
formatting fix

therealmark committed:
Formatting fix

therealmark committed:
Formatting fix

therealmark committed:
Update README.md

therealmark committed:
Update README.md

snarvaez reviewed (changes requested):
1. API key wording mentions naming the key as `demo` but the screen capture names as `liberate data ...`. 
2. Screen capture should be cropped to not show other browser tabs, etc.

therealmark committed:
Updated README.md

therealmark committed:
cropped out tabs in screenshot

therealmark committed:
updated screenshot with demo for api key description

therealmark commented:
@snarvaez i updated the README to specify the cluster size and MongoDB version. Also implemented your two suggestions about cropping out the browser tabs and renaming the API key to demo. One point to note is that two different API keys are now getting created. The one in the screenshot is for the realm-cli client which is tied to Data Services, and the other is the one for App Services (the original one). I don't know what causes more confusion. Two differently named API keys or a single name for both Data Services and App Services.

snarvaez reviewed (approved):
LGTM
